app/services/link/fetcher/fetcher.go

```diff
@@ -116,7 +116,16 @@ func (s *Fetcher) queueForItem(ctx context.Context, u url.URL, item datagraph.It
 func (s *Fetcher) ScrapeAndStore(ctx context.Context, u url.URL) (*link_ref.LinkRef, *scrape.WebContent, error) {
 	wc, err := s.sc.Scrape(ctx, u)
 	if err != nil {
-		return nil, nil, fault.Wrap(err, fctx.With(ctx))
+		s.logger.Warn("failed to scrape URL, storing link with basic information only",
+			slog.String("error", err.Error()),
+			slog.String("url", u.String()))
+
+		ln, err := s.lr.Store(ctx, u.String(), "", "", []link_writer.Option{}...)
```

Comment:
maybe the title can be the domain name of the URL, or the full URL... not sure what's better for UX here. Use best judgement for display on the UI.